std/logger/core.d

```diff
@@ -1566,8 +1567,15 @@ class StdForwardLogger : Logger
     {
         sharedLog = oldSharedLog;
     }
-    () @trusted { new Thread(() { log("foo"); }).start(); }();
+    Thread toWaitFor;
+    () @trusted { toWaitFor = new Thread(() { log("foo"); }).start(); }();
     log("bar");
+
+    () @trusted
+    {
+        Thread.sleep(dur!"msecs"(10));
```

Comment:
Why a sleep is required?

Reply:
It actually might not be required.

Thanks

Reply:
I think de should annotate the join as `@safe`, if we can. Just a note

Reply:
Unfortunately it is a virtual method, so adding `@safe` could break code that overrides it.